tpot/tpot.py

```diff
@@ -214,10 +214,10 @@ def fit(self, features, classes):
             np.random.shuffle(data_columns)
             training_testing_data = training_testing_data[data_columns]
 
-            training_indices, testing_indices = next(iter(StratifiedShuffleSplit(training_testing_data['class'].values,
-                                                                                 n_iter=1,
+            training_indices, testing_indices = train_test_split(training_testing_data.index,
```

Comment:
This doesn't look right. The call needs to look something like:

```python
(training_features, testing_features,
training_labels, testing_labels) = train_test_split(input_data.drop('class', axis=1).values, 
                                                    input_data['class'].values,
                                                    train_size=0.75, test_size=0.25,
                                                    random_state=RANDOM_STATE,
                                                    stratify=input_data['class'].values)
```

Have you tested this?